content/blog/2024-04-04-tmix-03.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "This Month in Xilem, March 2024"
+authors = ["Daniel McNab"]
++++
+
+Since the last update, there have been three newly released projects from Linebender:
+
+- [![Velato's version on crates.io.](https://img.shields.io/crates/v/velato.svg)](https://crates.io/crates/velato)
+ [Velato][]: A parser and driver for Lottie animation files in applications using Vello
+- [![Vello SVG's version on crates.io.](https://img.shields.io/crates/v/vello_svg.svg)](https://crates.io/crates/vello_svg)
+  [Vello SVG][]: Allows displaying (some) SVG documents in Vello applications
+- [![Tracing Android Trace's version on crates.io.](https://img.shields.io/crates/v/tracing_android_trace.svg)](https://crates.io/crates/tracing_android_trace)
+  [Android Trace][]: Resilient bindings to [NDK Tracing][] and an integration with [tracing][]
+
+<!-- TODO: Gif of Velato? -->
+
+Implementation work for [rfcs#1][] has started. Masonry is being ported to Vello in [masonry#56][], with the next step being a move to Winit.
+This work can be followed in the [#masonry][] stream on Zulip.
+
+We have landed some important improvements to Vello.
+Euler spiral based stroke expansion ([vello#496][]) enables GPU stroking by default, significantly improving performance in scenes which make heavy use of strokes.
+Additionally, we now support scenes with more than 64k draw objects ([vello#526]).
+We have also been making progress on [vello#366][], by evaluating different strategies for memory allocation.
+The first part of this has landed in [vello#537][], which avoids undefined behaviour in more out-of-memory cases.
+
+Additionally, we have been able to continue taking advantage of improvements in the [foundational font crates][fontations] from Google Fonts.
+This has enabled support for font fallback in [xilem#196][].
+Hinting has been unblocked by [vello#544][], although the implementation has not yet landed in Xilem.
+
+[rfcs#5][] has been approved, which will improve the robustness of our continuous integration.
+Kaur Kuut, the author of the RFC, has also landed these changes in [vello#523][].
+
+A focus for the next month is preparing for the [RustNL conference][] at the beginning of May.
+We are hoping to present a compelling demo there.
+
+[masonry#56]: https://github.com/PoignardAzur/masonry-rs/pull/56
+[Android Trace]: https://github.com/linebender/android_trace
+[Vello SVG]: https://github.com/linebender/vello_svg
+[Velato]: https://github.com/linebender/velato
```

Comment:
```suggestion
[Velato]: https://github.com/linebender/velato
[web demo of Velato]: https://linebender.org/velato/
```

Reply:
My one concern here is that this might not be a cool URL - if we ever have an actual page for Velato, that would be the probably URL. It frustrates me that <https://linebender.github.io/velato> redirects to this URL, when it would be *better* as the former, because we can be fairly sure that will be stable

Reply:
Yeah this URL is probably destined to rot. It's unfortunate but not the worst thing for the purposes of this post. The URL is likely to keep working as-is for a few months and when we do introduce an actual product page it will probably have some sort of link in it to the web demo. So not ideal, but better than a `404` or `DNS_PROBE_FINISHED_NXDOMAIN`.